nilearn/mass_univariate/tests/test_permuted_least_squares.py

```diff
@@ -443,12 +436,11 @@ def test_permuted_ols_with_covar_with_intercept_in_confonding_vars(
 
 
 def test_permuted_ols_with_multiple_constants_and_covars(
-    design, random_state=RANDOM_STATE
+    design, rng, random_state=RANDOM_STATE
```

Comment:
Do we still need random_state here ?

Reply:
OK I could remove this and also remove any use of RANDOM_STATE as a global in this test module.

But made realized the doc string of permuted_ols was not clear all types that random_state could take so I updated that.